crates/logic/src/connection/wrapper.rs

```diff
@@ -32,19 +33,19 @@ impl ConnectionWrapper {
     {
         self.link
             .send(ConnectionTask::Sync(Box::new(move |connection| {
-                if let Err(err) = connection.send_packet(packet, write_fn) {
-                    error!("Error when sending packet: {}", err);
-                }
+                connection.send_packet(packet, write_fn)?;
+
+                Ok(())
```

Comment:
Would this be more concise? (or did I miss something?)
```suggestion
                connection.send_packet(packet, write_fn)
```

Reply:
The `?` does some error conversion stuff here. It is necessary.

Reply:
And what about
```suggestion
                Ok(connection.send_packet(packet, write_fn)?)
```
?

Reply:
That would technically work, but it feels convoluted. Using the () specifically returned from that method feels weird.

Reply:
Really? I like how it makes everything one line instead of multiple. What exactly feels weird about it?